chcemvediet/apps/anonymization/management/commands/attachment_anonymization.py

```diff
@@ -0,0 +1,78 @@
+from optparse import make_option
+
+import magic
+from django.core.files.base import ContentFile
+from django.core.management.base import BaseCommand, CommandError
+
+from poleno.attachments.models import Attachment
+from poleno.utils.misc import squeeze
+from chcemvediet.apps.anonymization.models import AttachmentFinalization
+from chcemvediet.apps.inforequests.models import Action
+
+
+class Command(BaseCommand):
+    args = u'attachment [file]'
+    help = squeeze(u"""
+            Creates anonymization for the specified Attachment. By default, the file path is read
+            from stdin. File path can be explicitly passed as an command argument. Anonymization
+            created this way will be marked as successful. Only one successful anonymization can be
+            assigned to the Attachment.
+            """)
+
+    option_list = BaseCommand.option_list + (
+        make_option(u'--content_type',
+                    help=squeeze(u"""
+                            Content type of file, e.g. "application/pdf". Automatically computed if
+                            not specified.
+                            """)
+                    ),
+        make_option(u'--debug',
+                    default=u'',
+                    help=u'Debug message to the newly created anonymization. Empty by default.'
+                    ),
+        make_option(u'--force',
+                    action=u'store_true',
+                    help=squeeze(u"""
+                            The command refuses to anonymize attachment if a successful
+                            anonymization already exists. This flag disables this check. Deletes all
+                            existing successful anonymizations and creates new one. Unsuccessful
+                            anonymizations will stay unaffected.
+                            """)
+                    ),
+    )
+
+    def handle(self, *args, **options):
+        if not args:
+            raise CommandError(u'attachment_anonymization takes at least 1 argument (0 given).')
+        if len(args) > 2:
+            raise CommandError(
+                u'attachment_anonymization takes at most 2 arguments ({} given).'.format(len(args))
+                )
+        pk = args[0]
```

Comment:
@viliambalaz asi by bolo citatelnejsie premennu nazvat `attachment_pk`. Nech je citatelovi jasne, coho pk to je.

Reply:
:ok: